### ISSUE TITLE
E225 false positive on negative list indexes

### ISSUE THREAD

**Issue by GPHemsley** [PyCQA/pycodestyle#571]
I am getting an E225 (missing whitespace around operator) error on the following line:

```
parent = stack[-1]
```

This seems to me to be a false positive. It's a unary operator in this case, why should it require a space?

**Comment by daspecster** [comment]
@GPHemsley, I have a test.py file with the following code in it, based on your example.

```python
stack = []
parent = stack[-1]
```

I'm not getting any errors when running `pycodestyle test.py`.

What version of pycodestyle are you running? (my test was with 2.0.0)
If you could post a full example to test I can try again.

**Comment by GPHemsley** [comment]
You know what, maybe I had my line numbers wrong. I can't reproduce this myself.
